x-pack/plugins/ml/public/application/components/ml_inference/constants.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ADD_INFERENCE_PIPELINE_STEPS = {
+  DETAILS: 'Details',
+  CONFIGURE_PROCESSOR: 'Configure processor',
+  ON_FAILURE: 'Failure handling',
+  TEST: 'Test',
+  CREATE: 'create',
+} as const;
+
+export const CANCEL_BUTTON_LABEL = i18n.translate(
+  'xpack.ml.trainedModels.actions.cancelButtonLabel',
+  { defaultMessage: 'Cancel' }
+);
+
+export const CLOSE_BUTTON_LABEL = i18n.translate(
+  'xpack.ml.trainedModels.actions.closeButtonLabel',
+  { defaultMessage: 'Close' }
+);
+
+export const BACK_BUTTON_LABEL = i18n.translate('xpack.ml.trainedModels.actions.backButtonLabel', {
+  defaultMessage: 'Back',
+});
+
+export const CONTINUE_BUTTON_LABEL = i18n.translate(
+  'xpack.ml.trainedModels.actions.continueButtonLabel',
+  { defaultMessage: 'Continue' }
+);
+
+export const editMessage = i18n.translate(
```

Comment:
Nit: This file now has a mix of `camelCase` and `SNAKE_CASE`, would be good to stick to `SNAKE_CASE`.

Reply:
Good call - updated in b0aa5e0d8ecc48b474dcd709982536fe852dcd9a